extensions/import/src/wizard/dacfxExportWizard.ts

```diff
@@ -0,0 +1,127 @@
+'use strict';
+import * as vscode from 'vscode';
+import * as nls from 'vscode-nls';
+import * as sqlops from 'sqlops';
+import { ExportConfigPage } from './pages/exportConfigPage';
+import { DacFxDataModel } from './api/models';
+import { ExportSummaryPage } from './pages/exportSummaryPage';
+import { DacFxPage } from './api/dacFxPage';
+import { DacFxWizard } from './dacfxWizard';
+
+const localize = nls.loadMessageBundle();
+
+export class DacFxExportWizard extends DacFxWizard {
+	private wizard: sqlops.window.modelviewdialog.Wizard;
+	private connection: sqlops.connection.Connection;
+	private exportConfigPage: ExportConfigPage;
+	private summaryPage: ExportSummaryPage;
+	private model: DacFxDataModel;
+
+	constructor() {
+		super();
+	}
+
+	public async start(p: any, ...args: any[]) {
+		this.model = <DacFxDataModel>{};
+		let pages: Map<number, DacFxPage> = new Map<number, DacFxPage>();
+
+		let profile = p ? <sqlops.IConnectionProfile>p.connectionProfile : null;
+		if (profile) {
+			this.model.serverId = profile.id;
+			this.model.databaseName = profile.databaseName;
+		}
+
+		this.connection = await sqlops.connection.getCurrentConnection();
+		if (!this.connection) {
+			vscode.window.showErrorMessage(localize('dacFxExport.needConnection', 'Please connect to a server before using this wizard.'));
+			return;
+		}
+
+		this.wizard = sqlops.window.modelviewdialog.createWizard('Export Data-tier Application Wizard');
+		let page1 = sqlops.window.modelviewdialog.createWizardPage(localize('dacFxExport.page1Name', 'Export Settings'));
+		let page2 = sqlops.window.modelviewdialog.createWizardPage(localize('dacFxExport.page2Name', 'Summary'));
+
+		page1.registerContent(async (view) => {
+			this.exportConfigPage = new ExportConfigPage(this, page1, this.model, view);
+			pages.set(0, this.exportConfigPage);
+			await this.exportConfigPage.start().then(() => {
+				this.exportConfigPage.setupNavigationValidator();
+				this.exportConfigPage.onPageEnter();
+			});
+		});
+
+		page2.registerContent(async (view) => {
+			this.summaryPage = new ExportSummaryPage(this, page2, this.model, view);
+			pages.set(1, this.summaryPage);
+			await this.summaryPage.start();
+		});
+
+		this.wizard.onPageChanged(async (event) => {
+			let idx = event.newPage;
+			let page = pages.get(idx);
+
+			if (page) {
+				page.setupNavigationValidator();
+				page.onPageEnter();
+			}
+		});
+
+		this.wizard.onPageChanged(async (event) => {
+			let idx = event.lastPage;
+			let page = pages.get(idx);
+
+			if (page) {
+				page.onPageLeave();
+			}
+		});
+
+		this.wizard.pages = [page1, page2];
+		this.wizard.generateScriptButton.hidden = true;
+		this.wizard.doneButton.label = localize('dacFxExport.exportButton', 'Export');
+		this.wizard.doneButton.onClick(async () => await this.export());
+		this.wizard.open();
+	}
+
+	private async export() {
+		let connectionString = await this.getConnectionString();
```

Comment:
if possible I'd recommend using `ConnectionProfile` objects instead of connection strings.  Perhaps passing the connection URI and allow the service to find the related connection info, like we do with most features.